stories/dialogModal/index.stories.tsx

```diff
@@ -83,6 +83,82 @@ export const DefaultOpened = () => {
   );
 };
 
+export const WithColor = () => {
+  const [openModal, setOpenModal] = useState(false);
+
+  return (
+    <React.Fragment>
+      <Button onClick={() => setOpenModal(true)}>open dialog modal</Button>
+
+      {openModal && (
+        <DialogModal
+          title="Heads up!"
+          modalColor={primaryTheme.COLORS.background}
```

Comment:
Our stories are "theme-aware" as well, so I would pull the `theme` off of a `useTheme` invocation above the return for compatibility with the `secondaryTheme` in Storybook, otherwise it has the `primary` background:

<img width="458" alt="Screen Shot 2021-02-11 at 12 21 54 PM" src="https://user-images.githubusercontent.com/13544620/107680715-da011c00-6c63-11eb-8ac7-38ab8111e301.png">